Compilation error under arrach64

# QUESTION: <!-- Compilation error under arrach64-->

## Details

<!-- fill this out -->

<!--
How to solve the following error when compiling under arrach64，please!!!!

  -Iaarch64 -I.   -I../include -I../common/ -I../core/  -fPIC  ../../src/../src/libpmem2/aarch64/init.c
../../src/../src/libpmem2/aarch64/init.c: In function ‘pmem2_arch_init’:
../../src/../src/libpmem2/aarch64/init.c:88:3: error: implicit declaration of function ‘CORE_LOG_FATAL’ [-Werror=implicit-function-declaration]
   88 |   CORE_LOG_FATAL("invalid deep flush function address");
      |   ^~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [../Makefile.inc:288: ../nondebug/libpmem2/init.o] Error 1
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/root/pmdk/src/libpmem2'
make[1]: *** [Makefile:128: libpmem2] Error 2
make[1]: Target 'all' not remade because of errors.
make[1]: Leaving directory '/root/pmdk/src'
make: *** [Makefile:53: all] Error 2

-->

--- a/src/libpmem2/aarch64/init.c
+++ b/src/libpmem2/aarch64/init.c
@@ -7,6 +7,7 @@
 
 #include "auto_flush.h"
 #include "flush.h"
+#include "log_internal.h"
 #include "out.h"
 #include "pmem2_arch.h"
 

